packages/paste-core/components/pagination/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,212 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import type {PasteCustomCSS} from '@twilio-paste/customization';
+import {WrapperAndArrows, NumbersAndLabel} from '../stories/customization.stories';
+
+expect.extend(matchers);
+
+const paginationStyles = {fontSize: 'fontSize60', fontWeight: 'fontWeightBold'} as PasteCustomCSS;
+const getCustomizedStyles = (prefix = 'PAGINATION'): {[key: string]: any} => ({
+  [`${prefix}`]: {borderColor: 'colorBorder', borderStyle: 'solid', borderWidth: 'borderWidth10'},
+  [`${prefix}_ITEMS`]: {
+    borderColor: 'colorBorder',
+    borderStyle: 'solid',
+    borderWidth: 'borderWidth10',
+    margin: 'space30',
+  },
+  [`${prefix}_ITEMS_ITEM`]: {
+    borderColor: 'colorBorder',
+    borderStyle: 'solid',
+    borderWidth: 'borderWidth10',
+    margin: 'space30',
+  },
+  [`${prefix}_ARROW_BUTTON`]: {
+    backgroundColor: 'colorBackground',
+    borderColor: 'colorBorder',
+    borderStyle: 'solid',
+    borderWidth: 'borderWidth10',
+  },
+  [`${prefix}_ARROW_ICON_WRAPPER`]: {
+    padding: 'space40',
+    size: 'size10',
+  },
+  [`${prefix}_ARROW_ICON`]: {
+    size: 'size10',
+  },
+  [`${prefix}_ARROW_TEXT`]: {
+    fontWeight: 'fontWeightBold',
+    fontSize: 'fontSize50',
+  },
+  [`${prefix}_LABEL`]: paginationStyles,
+  [`${prefix}_ELLIPSIS`]: paginationStyles,
+  [`${prefix}_NUMBERS`]: paginationStyles,
+  [`${prefix}_NUMBERS_ITEM`]: paginationStyles,
+  [`${prefix}_NUMBER`]: paginationStyles,
+  [`${prefix}_NUMBER_TEXT`]: paginationStyles,
+});
+
+describe('Customization for Pagination wrapper and arrows', () => {
+  it('should set a element data attribute for the Pagination wrapper and arrows', () => {
+    const {getByTestId} = render(<WrapperAndArrows />);
+    const pagination = getByTestId('pagination');
+    const paginationItems = getByTestId('pagination-items');
+    const paginationBackArrow = getByTestId('pagination-back-arrow');
+
+    expect(pagination.getAttribute('data-paste-element')).toEqual('PAGINATION');
+
+    expect(paginationItems.getAttribute('data-paste-element')).toEqual('PAGINATION_ITEMS');
+    expect((paginationItems.firstChild as HTMLElement).getAttribute('data-paste-element')).toEqual(
+      'PAGINATION_ITEMS_ITEM'
```

Comment:
How do we feel about this items_item name? `PAGINATION_ITEMS_ITEM`.

I think that's the only thing that stands out to be, feels a bit cumbersome. Would you consider `PAGINATION_LIST` and `PAGINATION_LIST_ITEM`?

Reply:
I went back and forth on this. My hunch is that it's easier to reason about element names if they match the component name, so I went with this approach rather than changing it up.